MatrixSDKTests/TestPlans/UnitTests.xctestplan

```diff
@@ -13,15 +13,15 @@
       "targets" : [
         {
           "containerPath" : "container:MatrixSDK.xcodeproj",
-          "identifier" : "B14EF1C72397E90400758AF0",
-          "name" : "MatrixSDK-macOS"
+          "identifier" : "32C6F92C19DD814400EA4E9C",
+          "name" : "MatrixSDK-iOS"
```

Comment:
I may be missing some context here why we ran unit tests on macos instead of iOS. If it is important to keep as it was, I can instead add another test target with iOS-specific tests only. I will also try to package the MatrixCryptoSDK to run on macos so that this issue can be avoided altogether